## scan to map 参数
if_debug: true

# 坐标系
odom_frame: "odom"
base_frame: "base_footprint"
map_frame: "map"
lidar_frame: "base_scan"

# 匹配结果方差
Variance_X: 0.2
Variance_Y: 0.2
Variance_Yaw: 0.1

Use_TfTree_Always: true

# ICP定位参数
SCORE_THRESHOLD_MAX: 0.03      # 达到最大迭代次数后，代价仍高于此值，认为无法收敛
ANGLE_SPEED_THRESHOLD: 3      # 角速度阈值，角速度大于此值则不发送定位
ANGLE_THRESHOLD: 0.01         # 最小变换角度
DIST_THRESHOLD: 0.01          # 最小变换距离
ANGLE_UPPER_THRESHOLD: 0.06   # 最大角度变换
Point_Quantity_THRESHOLD: 125 # 点云数阈值

Maximum_Iterations: 80        # 最大迭代次数50

ObstacleRemoval_Distance_Max: 1.0     # 如果雷达点云中点在地图点云最近点大于此值，就认为该点为障碍点
VoxelGridRemoval_LeafSize: 0.1        # 体素滤波的边长

# 重定位参数
Relocation_Weight_Score: 1            # 重定位分数系数
Relocation_Weight_Distance: 1         # 重定位距离系数
Relocation_Weight_Yaw: 1              # 重定位yaw角系数

Relocation_Maximum_Iterations: 80     # 重定位最大迭代次数 80
Relocation_Score_Threshold_Max: 0.8   # 重定位代价阈值，低于此值认为正确重定位